tests/surveys/radio/results.json

```diff
@@ -0,0 +1,9 @@
+{
+    "count": 1,
+    "1": {
+        "1": 0,
+        "2": 0,
+        "3": 0,
+        "4": 1
+    }
+}
```

Comment:
Exactly as I expected :) Got a bit confused because I though this was meant to be a submission ^^